allofplos/gdrive.py

```diff
@@ -0,0 +1,156 @@
+import datetime
+import os
+import tarfile
+from zipfile import ZipFile, BadZipFile
+
+import requests
+from tqdm import tqdm
+
+from . import corpusdir
+
+# Variables needed
+zip_id = '0B_JDnoghFeEKLTlJT09IckMwOFk'
+metadata_id = '0B_JDnoghFeEKQUhKWXBOVy1aTlU'
+local_zip = 'allofplos_xml.zip'
+zip_metadata = 'zip_info.txt'
+time_formatting = "%Y_%b_%d_%Hh%Mm%Ss"
+min_files_for_valid_corpus = 200000
+test_zip_id = '12VomS72LdTI3aYn4cphYAShv13turbX3'
+local_test_zip = 'sample_corpus.zip'
+gdrive_url = "https://docs.google.com/uc?export=download"
+
+
+def download_file_from_google_drive(id, filename, destination=corpusdir,
```

Comment:
This entire file should be moved into its own PR as it is completely unrelated to the Corpus class. This should be merged before the corpus class though, since the Corpus class depends on it.